src/app/mytools/mytools.service.ts

```diff
@@ -57,13 +60,30 @@ export class MytoolsService extends MyEntriesService<DockstoreTool, OrgToolObjec
           this.alertService.detailedError(error);
         }
       );
+
+    this.usersService
+      .userAppTools(userId)
+      .pipe(finalize(() => this.myEntriesService.setRefreshingMyEntries(false)))
+      .subscribe((appTools: Array<Workflow>) => {
+        this.workflowService.setWorkflows(appTools);
+      });
   }
+
   selectEntry(tool: DockstoreTool | Workflow | null): void {
     if (tool && tool.id) {
-      this.containersService.getContainer(tool.id, includesValidation).subscribe((result) => {
-        this.location.go('/my-tools/' + result.tool_path);
-        this.containerService.setTool(result);
-      });
+      if (tool.path.includes('github.com')) {
```

Comment:
Suggest making a method out of this, like `isAppTool`. I had a hard time figuring out what this does at first.

You could go full bore and create a [user defined type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards), although the link shows as deprecated and TypeScript may have a new way of doing it.